src/oas3/transformers/request.ts

```diff
@@ -178,6 +178,9 @@ export const translateToRequest = withContext<
       kind = param.in;
     }
 
+    // TODO: Test with both path-level and op-level parameters!
```

Comment:
remove? Or is this still needing to be done?

Reply:
Done in e93cc80.